content/builds/hunter/liars-handshake.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Liars Handshake Hunter-Titan"
+description: "PvE build with Liar's Handshake"
+season: 15
+draft: false
+tags: ['hunter', 'pve', "liar's handshake"]
+d2_armor_picker_hash: 'n/a'
+dim_profile_link: 'https://app.destinyitemmanager.com/optimizer?class=1&p=%7B%22statConstraints%22%3A%5B%7B%22statHash%22%3A392767087%7D%2C%7B%22statHash%22%3A2996146975%7D%2C%7B%22statHash%22%3A1943323491%7D%2C%7B%22statHash%22%3A1735777505%7D%2C%7B%22statHash%22%3A144602215%7D%2C%7B%22statHash%22%3A4244567218%7D%5D%2C%22lockItemEnergyType%22%3Afalse%2C%22upgradeSpendTier%22%3A0%2C%22mods%22%3A%5B3470562293%2C1484685885%2C3730619869%2C4213142382%2C4044800077%2C1680735357%2C760472183%2C760472183%2C1513970148%2C1513970148%2C3065699930%2C3065699930%2C3836152936%2C2476831315%5D%2C%22exoticArmorHash%22%3A4165919945%7D'
+
+---
+
+## Overview
```

Comment:
Could we add the stats section back from the template? Newer players might find that info useful for adjusting the build to fit their existing armor.

Reply:
Oops, I saw the template but didn't copy from it when I made this. I'll bring the rest of the formatting up to date to match the template where it doesn't too.